hana/CHANGELOG.md

```diff
@@ -4,6 +4,13 @@
 - The format is based on [Keep a Changelog](http://keepachangelog.com/).
 - This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.2.1](https://github.com/cap-js/cds-dbs/compare/hana-v0.2.0...hana-v0.2.1) (2024-04-19)
+
+
+### Fixed
+
+* Use json datatype for Postgres insert ([#582](https://github.com/cap-js/cds-dbs/issues/582)) ([f1c9c89](https://github.com/cap-js/cds-dbs/commit/f1c9c89036a7f8e4709c67d713d06926630aa36d))
```

Comment:
If possible we should try to  avoid making changes in other DBs while fixing bugs for another. It is confusing that there is a "Postgres" note in the changelog for SAP HANA.